src/validateErrors.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+const errors = [
+  {
+    message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  },
+  {
+    message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+  },
+];
```

Comment:
Maybe better to use object for errors or even consts, because in code below hard to understand what error do you push.

Reply:
agree, not clear at all, high probability of mistake